storage/raft_transport.go

```diff
@@ -394,9 +394,13 @@ func (t *RaftTransport) SendAsync(req *RaftMessageRequest) bool {
 
 // SendSync sends a raft message and waits for an acknowledgement.
 func (t *RaftTransport) SendSync(ctx context.Context, req *RaftMessageRequest) error {
-	conn := t.getNodeConn(req.ToReplica.NodeID)
-	if conn == nil {
-		return errors.Errorf("unable to get connection for node %s", req.ToReplica.NodeID)
+	addr, err := t.resolver(req.ToReplica.NodeID)
```

Comment:
This is a fix, but not what I'm intending. I'd like to reuse the breaker the previous change introduced.